### DISCUSSION TITLE
Register 4 keys shortcuts

### DISCUSSION THREAD

**Discussion by fascox** [rxhanson/Rectangle#618] [Ideas]
3 modifier keys (ctrl+alt+cmd) plus key or cursor key.

**Comment by rxhanson** [comment]
I don't quite understand what you're asking for here. Can you provide an example?

**Reply by fascox** [reply]
yes with pleasure, this is a simple request. I would like to be able to activate shortcuts with a 3-key modifier (CMD-ALT-CTRL) . 
Now it doesn't seem possible.

**Reply by rxhanson** [reply]
My guess is that the shortcuts you are trying to record/execute are being used elsewhere or are already registered elsewhere with another app, and are not making it to Rectangle. You can try checking the box for removing shortcut restrictions in the prefs window and see if that makes any difference.

**Reply by fascox** [reply]
ok, i have tried the 3 key modifier combination with a letter/symbol and it register correctly, but when i try to register CTRL+ALT+CMD+"a cursor key" it won't. Can you confirm?

**Reply by rxhanson** [reply]
Perhaps you need to remove the shortcuts for next/prev display?

**Reply by fascox** [reply]
<img width="776" alt="Schermata 2021-11-03 alle 14 24 44" src="https://user-images.githubusercontent.com/700221/140068091-8712d7ac-d4ac-4d96-ad80-cd49da96659c.png">

have tried to remove shortcuts next/prev display as you suggested but still prevent me to register half left/right shortcut with CTRL+ALT+CMD+left or right cursor

even more strange it work when register with CTRL+ALT+CMD+SHIFT+left or right cursor
<img width="406" alt="Schermata 2021-11-03 alle 14 31 36" src="https://user-images.githubusercontent.com/700221/140069352-3902fdc2-04eb-4f2b-b745-3fc0553e9ddc.png">

**Reply by fascox** [reply]
ok it worked, some shortcut in keyboard settings. thank you.
